### PR TITLE
Hubert / Install setup linaria

### PR DESCRIPTION
**Pull request by hubert-deriv**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,9 @@
   "dependencies": {
     "@docusaurus/core": "2.2.0",
     "@docusaurus/preset-classic": "2.2.0",
+    "@linaria/babel-preset": "^4.3.1",
```
Comment (michio-rework) [inline package.json:21]: please move these to `devDependencies`, thanks thanks :pray:

Comment (michio-rework) [reply package.json:21]: `@linaria/babel-preset`, `@linaria/core` `@linaria/react`

File: package.json [hunk 2]
```diff
@@ -29,16 +32,24 @@
     "@babel/preset-react": "^7.18.6",
     "@babel/preset-typescript": "^7.18.6",
     "@docusaurus/module-type-aliases": "2.2.0",
+    "@linaria/webpack-loader": "^4.1.9",
```
Comment (michio-rework) [inline package.json:35]: please move this to `devDependencies`, thanks thanks :pray:

File: package.json [hunk 3]
```diff
@@ -29,16 +32,24 @@
...
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^12.1.5",
     "@testing-library/user-event": "^14.4.3",
     "@tsconfig/docusaurus": "^1.0.5",
     "babel-jest": "^29.3.1",
+    "babel-loader": "^9.1.0",
```
Comment (michio-rework) [inline package.json:41]: please remove `babel-loader`, `css-loader`, `file-loader`, `mini-css-extract-plugin`, `webpack`, `webpack-cli`, `webpack-dev-server`, we don't need them, thanks thanks :pray:

**Review by michio-rework — CHANGES_REQUESTED** [review]

**Comment by hubert-deriv** [comment]
Creating new pr here: https://github.com/binary-com/deriv-api-docs/pull/7

due to package lock merge conflicts.

**Pull request closed without merging**